### PR TITLE
shnsplit: fix typo, systemctl: update Korean translation, linux/s*: add Korean translation

### PR DESCRIPTION
**Pull request by CodePsy-2001**
- **linux/p*: add Korean translation**
- **linux/s*: add Korean translation**
- **systemctl: update Korean translation**

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by CodePsy-2001** [comment]
force-pushed: linux/p* branch were accidentally included and removed

File: pages.ko/linux/semanage.md
Comment (Zamoca42) [inline pages.ko/linux/semanage.md:1]: The translation format of the description needs to be modified.

File: pages.ko/linux/shar.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`shar --uuencode {{경로/대상/파일1 경로/대상/파일2 ...}} > {{경로/대상/아카이브.sh}}`
+
+- 모든 파일을 텍스트로 처리 (즉, 아무것도 `uuencode`하지 않음):
+
+`shar --text-files {{경로/대상/파일1 경로/대상/파일2 ...}} > {{경로/대상/아카이브.sh}}`
+
+- 아카이브의 헤더 주석에 이름과 컷 마크 포함:
+
+`shar --archive-name "{{내 파일}}" --cut-mark {{경로/대상/파일1 경로/대상/파일2 ...}} > {{경로/대상/아카이브.sh}}`
```
Comment (Zamoca42) [inline pages.ko/linux/shar.md:24]:
```suggestion
`shar --archive-name "{{내_파일}}" --cut-mark {{경로/대상/파일1 경로/대상/파일2 ...}} > {{경로/대상/아카이브.sh}}`
```

File: pages.ko/linux/sqfscat.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# sqfscat
+
+> squashfs 파일 시스템에서 파일을 연결하고 `stdout`에 출력.
+> 더 많은 정보: <https://manned.org/sqfscat>.
+
+- squashfs 파일 시스템에서 하나 이상의 파일 내용을 표시:
+
+`sqfscat {{파일시스템.squashfs}} {{파일1 파일2 ...}}`
```
Comment (Zamoca42) [inline pages.ko/linux/sqfscat.md:8]:
```suggestion
`sqfscat {{파일_시스템.squashfs}} {{파일1 파일2 ...}}`
```

File: pages.ko/linux/sqfstar.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`xzcat {{아카이브.tar.xz}} | sqfstar {{파일시스템.squashfs}} {{파일1 파일2 ...}}`
+
+- `zstd`로 압축된 tar 아카이브에서 `.gz`로 끝나는 파일을 제외하고 squashfs 파일 시스템 생성:
+
+`zstdcat {{아카이브.tar.zst}} | sqfstar {{파일시스템.squashfs}} "{{*.gz}}"`
+
+- `lz4`로 압축된 tar 아카이브에서 정규 표현식에 맞는 파일을 제외하고 squashfs 파일 시스템 생성:
+
+`lz4cat {{아카이브.tar.lz4}} | sqfstar {{파일시스템.squashfs}} -regex "{{정규_표현식}}"`
```
Comment (Zamoca42) [inline pages.ko/linux/sqfstar.md:24]:
```suggestion
`sqfstar {{파일_시스템.squashfs}} < {{아카이브.tar}}`

- `gzip`으로 압축된 tar 아카이브에서 특정 알고리즘으로 파일 시스템을 [comp]압축하여 squashfs 파일 시스템 생성:

`zcat {{아카이브.tar.gz}} | sqfstar -comp {{gzip|lzo|lz4|xz|zstd|lzma}} {{파일_시스템.squashfs}}`

- `xz`로 압축된 tar 아카이브에서 일부 파일을 제외하고 squashfs 파일 시스템 생성:

`xzcat {{아카이브.tar.xz}} | sqfstar {{파일_시스템.squashfs}} {{파일1 파일2 ...}}`

- `zstd`로 압축된 tar 아카이브에서 `.gz`로 끝나는 파일을 제외하고 squashfs 파일 시스템 생성:

`zstdcat {{아카이브.tar.zst}} | sqfstar {{파일_시스템.squashfs}} "{{*.gz}}"`

- `lz4`로 압축된 tar 아카이브에서 정규 표현식에 맞는 파일을 제외하고 squashfs 파일 시스템 생성:

`lz4cat {{아카이브.tar.lz4}} | sqfstar {{파일_시스템.squashfs}} -regex "{{정규_표현식}}"`
```

**Review by Zamoca42 — CHANGES_REQUESTED** [review]

**Review by Zamoca42 — APPROVED** [review]

**Pull request merged**